pkg/chartverifier/pyxis/pyxis_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pyxis
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func Test_getImageRegistries(t *testing.T) {
+
+	type testCase struct {
+		description string
+		repository  string
+		registry    string
+		message     string
+	}
+
+	PassTestCases := []testCase{
+		{description: "Test nginx respository", repository: "nginx", registry: "registry.hub.docker.com", message: ""},
+		{description: "Test rhel6.7 respository", repository: "rhel6.7", registry: "registry.access.redhat.com", message: ""},
+		{description: "Test rhel8/nginx-116 respository", repository: "rhel8/nginx-116", registry: "registry.access.redhat.com", message: ""},
+		{description: "Test ibm/nginx respository", repository: "ibm/nginx", registry: "non_registry", message: ""},
+		{description: "Test turbonomic/zookeeper respository", repository: "turbonomic/zookeeper", registry: "registry.connect.redhat.com", message: ""}}
+
+	for _, tc := range PassTestCases {
+		t.Run(tc.description, func(t *testing.T) {
+			reg, err := GetImageRegistries(tc.repository)
+			require.NoError(t, err)
+			require.Equal(t, tc.registry, reg[0])
+		})
+	}
+
+	FailTestCases := []testCase{
+		{description: "Test repository not found", repository: "ndoesnotexist", registry: "registry.hub.docker.com", message: "Respository not found"},
+	}
+
+	for _, tc := range FailTestCases {
+		t.Run(tc.description, func(t *testing.T) {
+			reg, err := GetImageRegistries(tc.repository)
+			require.Error(t, err)
+			require.Empty(t, reg)
+			require.Contains(t, err.Error(), tc.message)
+		})
+	}
+
+}
+
+func Test_checkImageInRegistry(t *testing.T) {
+
+	type testCase struct {
+		description string
+		repository  string
+		registry    string
+		version     string
+		message     string
+	}
+
+	PassTestCases := []testCase{
+		{description: "Test nginx registry and version", repository: "nginx", registry: "registry.hub.docker.com", version: "latest", message: ""},
```

Comment:
since all tests are verifying that images are certified, how about to state that in test names? btw, we also need negative cases as well.

Reply:
Updated to Test nginx registry and version found."